components/script/dom/webglrenderingcontext.rs

```diff
@@ -3612,7 +3615,12 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
     ) -> JSVal {
         // FIXME(nox): https://github.com/servo/servo/issues/21133
 
-        if program.is_deleted() || !program.is_linked() || program.id() != location.program_id() {
+        if
+            program.is_deleted() ||
+            !program.is_linked() ||
+            program.id() != location.program_id() ||
```

Comment:
Maybe worth moving this into a `UniformLocation::linked_in(&WebGLProgram)` or something like that?

Reply:
Yeah ultimately I want to refactor all the things but I'm not sure where it should go and how it should like for now.